organizations/models.go

```diff
@@ -172,3 +174,18 @@ type ChatSettings struct {
 	MediaVisibility bool   `json:"media_visibility" bson:"media_visibility"`
 	FontSize        string `json:"font_size" bson:"font_size"`
 }
+
+
+// gets the details of a member in a workspace using parameters such as email, username etc
+// returns parameters based on the member struct
+func FetchMember(filter map[string]interface{}) (*Member, error) {
```

Comment:
What advantage does this have over the GetMongoDbDoc function?

Reply:
Hmmmm


This one.... Me I cannot answer. Looking at my screen right now and thinking why I no try this one out first.

I'll try this out, it should give me the same thing

Reply:
Hmmmm


This one.... Me I cannot answer. Looking at my screen right now and thinking why I no try this one out first.

I'll try this out, it should give me the same thing

Reply:
orgDoc, _ := utils.GetMongoDbDoc(OrganizationCollectionName, bson.M{"_id": orgId})
	if orgDoc == nil {
		utils.GetError(errors.New("organization does not exist"), http.StatusBadRequest, w)
		return
	}

this returns just Primitive.M...checks if a document exists based on the parameters supplied. Does not return a set of data based on the member struct like member.ID, member.Email etc